src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/DefaultWindowsRazorProjectHostTest.cs

```diff
@@ -749,8 +749,60 @@ public async Task OnProjectChanged_ReadsProperties_InitializesProject()
         Assert.Empty(_projectManager.GetProjects());
     }
 
+    [UITheory]
+    // Standard setup.  BaseIntermediateOutputPath ends in obj and IntermediateOutputPath starts with obj
+    [InlineData("C:\\my repo root\\solution folder\\projectFolder\\obj\\", "obj\\Debug\\net8.0", "C:\\my repo root\\solution folder\\projectFolder\\obj\\Debug\\net8.0")]
```

Comment:
```suggestion
    [InlineData(@"C:\my repo root\solution folder\projectFolder\obj\", @"obj\Debug\net8.0", @"C:\my repo root\solution folder\projectFolder\obj\Debug\net8.0")]
```

Reply:
and others below

Reply:
Yes please! Verbatim strings have been in the language since C# 1.0

Reply:
Done